client/client.go

```diff
@@ -35,10 +35,14 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
-func unixDialer(_, _ string) (net.Conn, error) {
+func privateUnixDialer(_, _ string) (net.Conn, error) {
```

Comment:
> I don't think we share a common meaning for the "private" word. :-) How is it private? Every single snap will see this socket. Every single user in the system will see this socket. World readable, and writable. Hard for me to empathize with the term.

Using the perspective from within the snap, snaps can't actually see the `snapd.socket`. It's denied by apparmor unless you use the `snapd-control` interface, which to me is requesting special permission to obtain access to something which is typically not visible (so perhaps "hidden" would have been a better term). Of course, from the overall system perspective (say I was SSHd in), of course it's not private or hidden.

> Yeah, let's rename that way then. Seems more clear indeed.

Alright, will do.